components/ui/button.vue

```diff
@@ -24,6 +24,10 @@
     </template>
 
     {{ label }}
+
+    <template v-if="chevron">
```

Comment:
Why would a button normally have a chevron?
Either change the content of the button to be a slot (as would be the case if #30 gets merged), or add another slot just for the icon (named append-icon) so that it can be as generic as possible.

Reply:
Well, this is not about the icon itself. Chevron has its own class with a transition applied, so each click on the button would rotate the chevron. It is useful